.github/workflows/ci.yaml

```diff
@@ -101,6 +101,7 @@ jobs:
 
           # Run the NLP task
           docker compose run --rm \
+            --pull never \
```

Comment:
This issue got introduced when I added `pull_policy: always` to the compose.yaml file, to force most folks to always get the latest image. But here, it's working against us since I want to use the just-built testing image. So override the pull policy from the CLI.